pkg/constants/constants.go

```diff
@@ -24,6 +24,12 @@ const (
 	// TODO(#23): Use the kubernetes.io domain when graduating APIs to beta.
 	QueueAnnotation = "kueue.x-k8s.io/queue-name"
 
+	// IgnoreAnnotation is the annotation used to mark workloads to be ignored
+	// by Kueue for scheduling, when the value of the annotation is `true`.
+	//
+	// TODO(#23): Use the kubernetes.io domain when graduating APIs to beta.
+	IgnoreAnnotation = "kueue.x-k8s.io/ignore"
```

Comment:
Can you consider this?

```suggestion
	ParentWorkloadAnnotation = "kueue.x-k8s.io/parent-workload"
```

And the value would be the name of the workload. Otherwise a user could abuse the `ignore` annotation and add it to any job to run immediately.

This gives us the chance to validate that the Workload is indeed admitted and even suspend the job if it's not. It could even simplify a custom job implementation that only uses Jobs: the CRD itself wouldn't even need a suspend field. Kueue suspends the underlying jobs.

And we probably need a `podset-name` annotation.

Note that for now we still need the suspend field in MPIJob, because the worker pods are not under a job.

cc @mwielgus

Reply:
Ok, I think I like the approach, I will probably prepare a new PR with that as the approach seems different enough. I'm wondering yet why we need the `podset-name` annotation?

Reply:
Having the podset-name would allow us to verify that the job matches the workload's podset spec, and suspend the job if it doesn't match. It would be a safety mechanism for rogue users that are trying to bypass kueue. But we can leave that for the future.